VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public const string Password = "abc";
```

Comment:
**CODE_SMELL**
> 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false)

Reply:
/workItem

Reply:
/workItem

Reply:
/workItem

Reply:
/workItem

Reply:
/workItem

Reply:
WorkItem 808 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/808).

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
The issue is already resolved or confirmed.